<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QMainWindow" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dyngine Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="frameBuffer">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionProject"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionEditor_Settings"/>
    <addaction name="actionProject_Settings"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuFind">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="title">
      <string>Find</string>
     </property>
     <addaction name="actionFind_2"/>
     <addaction name="actionReplace"/>
     <addaction name="separator"/>
     <addaction name="actionFind_in_files"/>
     <addaction name="actionReplace_in_Files"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="menuFind"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuEnter">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="title">
      <string>Enter...</string>
     </property>
     <addaction name="actionPresentation_Mode"/>
    </widget>
    <addaction name="actionTool_Windows"/>
    <addaction name="menuEnter"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>Build</string>
    </property>
    <widget class="QMenu" name="menuBuild_2">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="title">
      <string>Build...</string>
     </property>
     <addaction name="actionShaders"/>
     <addaction name="actionEngine_Content"/>
    </widget>
    <addaction name="menuBuild_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuBuild"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QDockWidget" name="treeFileExplorerDock">
   <property name="windowTitle">
    <string>Project Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="treeExplorerDocketContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="treeFileExplorer"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="tileFileExplorerDock">
   <property name="windowTitle">
    <string>File Content Viewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="tileExplorerDockContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListView" name="listFileExplorer">
       <property name="movement">
        <enum>QListView::Snap</enum>
       </property>
       <property name="flow">
        <enum>QListView::LeftToRight</enum>
       </property>
       <property name="gridSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionProject">
   <property name="text">
    <string>Project</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionEditor_Settings">
   <property name="text">
    <string>Editor Settings</string>
   </property>
  </action>
  <action name="actionProject_Settings">
   <property name="text">
    <string>Project Configuration</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionFind_in_path">
   <property name="text">
    <string>Find in path</string>
   </property>
  </action>
  <action name="actionFind_2">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionReplace_in_Files">
   <property name="text">
    <string>Replace in Files</string>
   </property>
  </action>
  <action name="actionTool_Windows">
   <property name="text">
    <string>Tool Windows</string>
   </property>
  </action>
  <action name="actionPresentation_Mode">
   <property name="text">
    <string>Presentation Mode</string>
   </property>
  </action>
  <action name="actionShaders">
   <property name="text">
    <string>Shaders</string>
   </property>
  </action>
  <action name="actionEngine_Content">
   <property name="text">
    <string>Engine Content</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFind_in_files">
   <property name="text">
    <string>Find in files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
